index.js

```diff
@@ -158,17 +166,17 @@ module.exports = {
     "func-name-matching": "error",
     "func-names": "error",
     "id-length": ["error", { min: 1, max: 30 }],
-    indent: ["error", 2, { SwitchCase: 1 }],
+    "indent": ["error", 2, { SwitchCase: 1 }],
     "jsx-quotes": ["error", "prefer-double"],
     "key-spacing": "error",
     "keyword-spacing": "error",
     "line-comment-position": "error",
     "linebreak-style": "error",
     "lines-between-class-members": "error",
     "max-depth": "error",
-    "max-lines": "error", // NOTE: Disable Node
+    "max-lines": "error",
     "max-nested-callbacks": ["error", { max: 3 }],
-    "max-params": ["error", { max: 4 }],
+    "max-params": ["error", 4],
```

Comment:
Although in the example in docs it's shown like this, the doc says it should be used as { "max": 4 }. Are you sure this works? Link to docs: https://eslint.org/docs/rules/max-params

Reply:
Yes I am. I tested that rule and works ok. In Node we use that approach so I moved it here to maintain all rules syntax updated.